heat/core/arithmetics.py

```diff
@@ -107,32 +107,51 @@ def diff(a, n=1, axis=-1):
     size = a.comm.size
     rank = a.comm.rank
     ret = a.copy()
-    for _ in range(n):  # work loop, runs n times. using the result at the end of the loop as the starting values for each loop
+    for _ in range(
+        n
+    ):  # work loop, runs n times. using the result at the end of the loop as the starting values for each loop
         axis_slice = [slice(None)] * len(ret.shape)
         axis_slice[axis] = slice(1, None, None)
         axis_slice_end = [slice(None)] * len(ret.shape)
         axis_slice_end[axis] = slice(None, -1, None)
 
         arb_slice = [slice(None)] * len(a.shape)
-        arb_slice[axis] = 0  # build the slice for the first element on the specified axis
+        arb_slice[
```

Comment:
This also seems wrong, comment should be above

Reply:
It is only made a multi-line statement, because the line is too long including the comment. Putting the comment in the line before the statement, Black happily accepts it as (and transforms it back to) a single line statement.